.ado/azure-pipelines.publish.yml

```diff
@@ -12,27 +11,69 @@ pr: none
 variables:
   - group: 'NPM and Github Secrets'
 
-pool:
-  vmImage: 'ubuntu-latest'
-
-steps:
-  - task: NodeTool@0
-    inputs:
-      versionSpec: '12.x'
-    displayName: 'Install Node.js'
-
-  - script: |
-      git config user.name "UI-Fabric-RN-Bot"
-      git config user.email "uifrnbot@microsoft.com"
-      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/ui-fabric-react-native.git
-    displayName: Git Authentication
-  - template: templates/yarn-install.yml
-
-  - script: |
-      yarn test
-    displayName: 'yarn test'
-  - script: |
-      yarn publish:beachball -- -b origin/master -n $(npmAuth) --access public -y
-    displayName: 'Publish NPM Packages'
-
-  - template: templates/publish-nuget-job.yml
+jobs:
+  - job: NPMPublish
+    displayName: NPM Publish
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '12.x'
+        displayName: 'Install Node.js'
+
+      - script: |
+          git config user.name "UI-Fabric-RN-Bot"
+          git config user.email "uifrnbot@microsoft.com"
+          git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/ui-fabric-react-native.git
+        displayName: Git Authentication
+      - template: templates/yarn-install.yml
+
+      - script: |
+          yarn test
+        displayName: 'yarn test'
+
+      - script: |
+          yarn publish:beachball -- -b origin/master -n $(npmAuth) --access public -y
+        displayName: 'Publish NPM Packages'
+
+  - job: NuGetPublish
+    displayName: NuGet Publish
```

Comment:
I might consider throwing the entire job into a template. Could be useful if we ever need to run test publish or decide to split things out into multiple pipelines

Reply:
Would you like me to move both NPM Publish and NuGet Publish to two separate templates? I can definitely see the usefulness in that

Reply:
I'll leave this for a future change that needs it, as this is already a lot of refactoring and I don't want to unnecessarily add more on top of that with this change